src/main/java/uk/ac/ebi/embl/template/CSVReader.java

```diff
@@ -94,26 +100,23 @@ private void prepareLineForParsing() {
     }
 
     private void readHeader(final int expectedMatchNumber, final List<TemplateTokenInfo> allTokens) throws Exception {
-        currentLine = readLine();
-        if (currentLine == null)
+        readLine();
+        if (currentLine == null) {
             throw new TemplateException("Template file is empty");
+        }
         String header = null;
-        boolean headerFound = false;
-        while (currentLine != null) {
-            currentLine = currentLine.replaceFirst("\"", "");//get rid of starting " if present - open office puts these in for strings and they need to be removed
-            if (currentLine.startsWith(CSVWriter.HEADER_TOKEN)) {
-                header = currentLine;
-                headerFound = true;
-                break;
-            }
-            currentLine = readLine();
+        skipEmptyLinesAndComments();
+        if (currentLine != null) {
+            header = currentLine;
+            readLine();
```

Comment:
First non comment line is taken as header line. This is good -> I think we must convey this to the user in the portal text.